dev-docs/bidders/adyoulike.md

```diff
@@ -11,17 +11,18 @@ gdpr_supported: true
 gvl_id: 259
 usp_supported: true
 floors_supported: true
+schain_supported: true
```

Comment:
Thanks for updating the docs. Could you add the flag multiformat_supported (introduced in https://github.com/prebid/prebid.github.io/pull/3694 ) as well?

It allows sales houses / publishers to better understand the behaviour of the bid adapter for multi format ad units.

```
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
```